packages/rocketchat-file-upload/client/lib/FileUploadGoogleStorage.js

```diff
@@ -0,0 +1,65 @@
+/* globals FileUpload, FileUploadBase, Slingshot */
+
+FileUpload.GoogleCloudStorage = class FileUploadGoogleCloudStorage extends FileUploadBase {
+	constructor(meta, file) {
+		super(meta, file);
+		this.uploader = new Slingshot.Upload('rocketchat-uploads', { rid: meta.rid });
+	}
+
+	start() {
+		this.uploader.send(this.file, (error, downloadUrl) => {
+			if (this.computation) {
+				this.computation.stop();
+			}
+
+			if (error) {
+				let uploading = Session.get('uploading');
+				if (!Array.isArray(uploading)) {
+					uploading = [];
+				}
+
+				const item = _.findWhere(uploading, {
+					id: this.id
+				});
+				if (_.isObject(item)) {
```

Comment:
This may just be my personal preference, but seems easier to read with a space before the if, and after the else below